pkg/destroy/powervs/powervs.go

```diff
@@ -429,41 +452,31 @@ func (o *ClusterUninstaller) loadSDKServices() error {
 		return fmt.Errorf("loadSDKServices: bxsession.New: %v", err)
 	}
```

Comment:
```suggestion
	}
	o.Logger.Debugf("loadSDKServices: bxSession = %v", bxSession)
```
I'm curious: why logging it once here is not enough?